### DISCUSSION TITLE
About Edgeless Mode

### DISCUSSION THREAD

**Discussion by Ding-Fan** [toeverything/AFFiNE#2713] [General]
- **Persistency of Blank Notes:** When I attempt to place four or five notes on the canvas to fill in gradually, I've noticed that I must first input some content, such as a space character, in each note to keep it visible. This requirement is quite distracting and hinders my thought process.

- **Connector Color in Dark Mode:** While using Affine in dark mode, the default color of the "connector" that links notes is black. This makes it difficult to see, and I have to manually adjust the color each time I draw a connector.

- **Connector Alignment with Moved Notes:** When drawing a connector between two notes, say "Note A" and "Note B", the connector accurately follows Note A if it's relocated. However, when I move Note B, the connector (specifically the arrow part) doesn't adjust its position accordingly. I must move Note A once more to realign the arrow with Note B, which can be quite inconvenient.

- **Lag with Multiple Connectors:** When moving a note that has several connectors attached to it, the software occasionally becomes laggy, particularly if I move my mouse swiftly. This issue could impact the overall usability and efficiency of the software.

**Comment by houjoe0829** [comment]
For the first, we still need to discuss the first question internally and keep an eye out for similar feedback.

For the second, we have planned to address this issue in the current iteration, and will follow up on it in this task.  https://github.com/toeverything/AFFiNE-Design/issues/13

**Comment by houjoe0829** [comment]
Regarding the third point, the main issue at the moment may be the lack of a connection indicator, which makes it unclear whether the connection has been established. We are adding an indicator to the connector to address this issue, and you can click here to keep track of our progress.  https://github.com/toeverything/blocksuite/issues/2654

**Comment by houjoe0829** [comment]
And regarding the fourth point, would it be possible for you to record a screencast for us to review, and provide your system version? Thank you again.

**Reply by Ding-Fan** [reply]
https://www.veed.io/view/75a29a62-8c1d-4bbb-9dd7-64c52ecb6a36?sharingWidget=true&panel=

OS: Pop!_OS 22.04 LTS x86_64

**Reply by houjoe0829** [reply]
It seems that this video has been archived.

**Reply by Ding-Fan** [reply]
Sorry the video host platform will delete video in 24 hrs, I found another platform will delete in 2 days, here is the link:
https://streamable.com/8smatl

**Reply by houjoe0829** [reply]
Got it, thanks again. We will continue to optimize the performance of the whiteboard in the future.
